[2019-09-25 16:01:18,157] INFO [GroupCoordinator 0]: Member consumer-2-952af6c3-3a4c-413d-8c84-7a9a00f4c72b in group customer-service has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:01:18,158] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 99 (__consumer_offsets-44) (reason: removing member consumer-2-952af6c3-3a4c-413d-8c84-7a9a00f4c72b on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:01:18,158] INFO [GroupCoordinator 0]: Group customer-service with generation 100 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:01:25,542] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 100 (__consumer_offsets-44) (reason: Adding new member consumer-2-8ff77fb8-750d-41d4-bc34-54eddca62138 with group instanceid None) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:01:25,543] INFO [GroupCoordinator 0]: Stabilized group customer-service generation 101 (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:01:25,547] INFO [GroupCoordinator 0]: Assignment received from leader for group customer-service for generation 101 (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:04:26,373] INFO [GroupCoordinator 0]: Member consumer-2-8ff77fb8-750d-41d4-bc34-54eddca62138 in group customer-service has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:04:26,374] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 101 (__consumer_offsets-44) (reason: removing member consumer-2-8ff77fb8-750d-41d4-bc34-54eddca62138 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:04:26,374] INFO [GroupCoordinator 0]: Group customer-service with generation 102 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:08:37,123] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-09-25 16:18:37,141] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-09-25 16:23:05,294] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 102 (__consumer_offsets-44) (reason: Adding new member consumer-2-fb54ce88-1a52-4a7f-8f35-01138c6766bd with group instanceid None) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:23:05,295] INFO [GroupCoordinator 0]: Stabilized group customer-service generation 103 (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:23:05,299] INFO [GroupCoordinator 0]: Assignment received from leader for group customer-service for generation 103 (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:28:37,159] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-09-25 16:38:37,153] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-09-25 16:39:25,852] INFO [GroupCoordinator 0]: Member consumer-2-fb54ce88-1a52-4a7f-8f35-01138c6766bd in group customer-service has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:39:25,853] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 103 (__consumer_offsets-44) (reason: removing member consumer-2-fb54ce88-1a52-4a7f-8f35-01138c6766bd on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:39:25,853] INFO [GroupCoordinator 0]: Group customer-service with generation 104 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:39:43,715] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 104 (__consumer_offsets-44) (reason: Adding new member consumer-2-aed11214-05ff-490b-af48-0cb83e303012 with group instanceid None) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:39:43,718] INFO [GroupCoordinator 0]: Stabilized group customer-service generation 105 (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:39:43,719] INFO [GroupCoordinator 0]: Assignment received from leader for group customer-service for generation 105 (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:40:26,625] INFO [GroupCoordinator 0]: Member consumer-2-aed11214-05ff-490b-af48-0cb83e303012 in group customer-service has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:40:26,626] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 105 (__consumer_offsets-44) (reason: removing member consumer-2-aed11214-05ff-490b-af48-0cb83e303012 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:40:26,626] INFO [GroupCoordinator 0]: Group customer-service with generation 106 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:40:36,199] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 106 (__consumer_offsets-44) (reason: Adding new member consumer-2-75dfc70c-c2b3-4c7f-a489-72b24c93276a with group instanceid None) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:40:36,200] INFO [GroupCoordinator 0]: Stabilized group customer-service generation 107 (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:40:36,210] INFO [GroupCoordinator 0]: Assignment received from leader for group customer-service for generation 107 (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:41:19,608] INFO [GroupCoordinator 0]: Member consumer-2-75dfc70c-c2b3-4c7f-a489-72b24c93276a in group customer-service has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:41:19,609] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 107 (__consumer_offsets-44) (reason: removing member consumer-2-75dfc70c-c2b3-4c7f-a489-72b24c93276a on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:41:19,609] INFO [GroupCoordinator 0]: Group customer-service with generation 108 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:41:28,562] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 108 (__consumer_offsets-44) (reason: Adding new member consumer-2-dfa8841a-4189-4e75-ae9a-5dbc45a0a9e9 with group instanceid None) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:41:28,563] INFO [GroupCoordinator 0]: Stabilized group customer-service generation 109 (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:41:28,567] INFO [GroupCoordinator 0]: Assignment received from leader for group customer-service for generation 109 (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:42:41,194] INFO [GroupCoordinator 0]: Member consumer-2-dfa8841a-4189-4e75-ae9a-5dbc45a0a9e9 in group customer-service has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:42:41,194] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 109 (__consumer_offsets-44) (reason: removing member consumer-2-dfa8841a-4189-4e75-ae9a-5dbc45a0a9e9 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:42:41,194] INFO [GroupCoordinator 0]: Group customer-service with generation 110 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:42:47,379] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 110 (__consumer_offsets-44) (reason: Adding new member consumer-2-614c1674-0ddc-4cdf-a1d7-01dfb5f8a777 with group instanceid None) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:42:47,379] INFO [GroupCoordinator 0]: Stabilized group customer-service generation 111 (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:42:47,381] INFO [GroupCoordinator 0]: Assignment received from leader for group customer-service for generation 111 (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:48:37,166] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-09-25 16:49:20,017] INFO [GroupCoordinator 0]: Member consumer-2-614c1674-0ddc-4cdf-a1d7-01dfb5f8a777 in group customer-service has left, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:49:20,017] INFO [GroupCoordinator 0]: Preparing to rebalance group customer-service in state PreparingRebalance with old generation 111 (__consumer_offsets-44) (reason: removing member consumer-2-614c1674-0ddc-4cdf-a1d7-01dfb5f8a777 on LeaveGroup) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:49:20,017] INFO [GroupCoordinator 0]: Group customer-service with generation 112 is now empty (__consumer_offsets-44) (kafka.coordinator.group.GroupCoordinator)
[2019-09-25 16:58:37,182] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-09-25 16:58:56,837] INFO Unable to read additional data from server sessionid 0x1001e5c81b30000, likely server has closed socket, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:58:58,086] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:58:58,086] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:58:59,947] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:58:59,947] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:01,833] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:01,833] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:03,354] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:03,354] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:05,407] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:05,408] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:07,289] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:07,289] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:08,449] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:08,449] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:10,324] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:10,324] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:12,127] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:12,128] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:14,057] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:14,057] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:15,291] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:15,291] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:16,678] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:16,678] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:18,499] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:18,499] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:19,873] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:19,874] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:21,873] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:21,874] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:23,759] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:23,759] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:25,305] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:25,306] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:27,348] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:27,349] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:28,713] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:28,713] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:30,775] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:30,775] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:30,871] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:30,874] INFO [KafkaServer id=0] shutting down (kafka.server.KafkaServer)
[2019-09-25 16:59:30,876] INFO [KafkaServer id=0] Starting controlled shutdown (kafka.server.KafkaServer)
[2019-09-25 16:59:32,099] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:32,099] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:32,208] INFO [ZooKeeperClient Kafka server] Waiting until connected. (kafka.zookeeper.ZooKeeperClient)
[2019-09-25 16:59:33,768] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:33,769] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:35,482] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:35,482] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:36,168] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:36,584] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:36,730] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:36,865] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:37,057] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:37,488] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:37,488] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:39,132] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:39,132] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:40,752] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:40,753] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:42,257] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:42,258] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:43,640] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:43,642] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:45,240] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:45,241] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:46,838] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:46,838] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:48,524] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:48,525] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:49,911] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:49,912] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:51,240] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:51,241] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:53,198] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:53,199] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:54,431] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:54,432] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:55,147] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:55,462] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:55,721] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:55,979] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:56,181] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:56,193] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:56,193] INFO Socket error occurred: localhost/127.0.0.1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:56,407] INFO Terminating process due to signal SIGINT (org.apache.kafka.common.utils.LoggingSignalHandler)
[2019-09-25 16:59:57,359] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:57,359] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:59,404] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-09-25 16:59:59,404] INFO Socket error occurred: localhost/0:0:0:0:0:0:0:1:2181: Connection refused (org.apache.zookeeper.ClientCnxn)
